.github/workflows/dapps.yml

```diff
@@ -438,7 +437,7 @@ jobs:
 
   destroy:
     runs-on: ubuntu-20.04
-    needs: [prepare, uniswap, saddle, curve, aave, swap-report]
+    needs: [prepare, uniswap, saddle, curve, aave, yearn]
```

Comment:
we need swap report too, because we get information from blockchain about gas and accounts